common/src/main/java/apoc/result/VirtualPath.java

```diff
@@ -139,23 +139,4 @@ private void requireConnected(Relationship relationship) {
             throw new IllegalArgumentException("Relationship is not part of current path.");
         }
     }
-    
-    public static final class Builder {
```

Comment:
I prefer to have it in there, now it's used only in `DataVirtualizationCatalog` but by design would be better having it in there; as in the future could be used in other use-cases